File: portfolio-BackEnd/src/main/java/com/portfolio/ag/Entity/WorkExp.java
package com.portfolio.ag.Entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class WorkExp {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String workName;
    private String workInfo;
    
    //Constructors
    public WorkExp() {
    }

    public WorkExp(String workName, String workInfo) {
        this.workName = workName;
        this.workInfo = workInfo;
    }
    
    //Getters - setters
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getWorkName() {
        return workName;
    }

    public void setWorkName(String workName) {
        this.workName = workName;
    }

    public String getWorkInfo() {
        return workInfo;
    }

    public void setWorkInfo(String workInfo) {
        this.workInfo = workInfo;
    }
    
}

File: src/app/model/work-exp.ts
export class WorkExp {
    id?: number;
    workName: string;
    workInfo: string;
    workDate: string;
    workImg: string;

    constructor (workName: string, workInfo: string, workDate: string, workImg: string) {
        this.workName = workName;
        this.workInfo = workInfo;
        this.workDate = workDate;
        this.workImg = workImg;
    }
}